experimental/dds/tree/src/SharedTree.ts

```diff
@@ -100,6 +100,7 @@ import {
 	ghostSessionId,
 	reservedIdCount,
 } from './persisted-types/index.js';
+import { SharedTreeAttributes, SharedTreeType } from './publicContracts.js';
```

Comment:
Do we need attributes, or just the type? Looking at implemenantation of DelayLoadFactory in [this PR](https://dev.azure.com/office/OC/_git/office-bohemia/pullrequest/3736156?path=/packages/fluid-extension/src/delayLoadDDSUtils/DelayLoadFactory.ts&_a=files), it sounds like type is all we need, no?

Reply:
We just need attributes. Type was part of it, so I exported that as well.

Reply:
Hm, IChannelFactory requires attributes, which feels wrong (it feels like factory should be able to make runtime decision on snapshotFormatVersion). But it's too expensive to change, so probably going with attributes is OK